NPM doesn't install assets referenced in lib/constants

In the latest release(s) (smooch@3.0.x), installing the library through npm only pulls the `lib` directory, but fails to pull the assets and images directories. This causes the following kind of errors on build (using gulp + browserify):

```
Error: Cannot find module '../../images/telegram_large_2x.png' from '<your/node/project>/node_modules/smooch/lib/constants'
```

Please let me know if you would like any additional information. Cheers

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
     "hot-dev-server": "webpack-dev-server --config webpack-hot-dev-server.config.js --hot --colors --port 2992 --inline --no-info",
     "build": "webpack --config webpack-production.config.js --progress --profile --colors",
     "build:assets": "webpack --config webpack-assets.config.js --progress --profile --colors",
-    "build:npm": "rm -rf lib/ && npm run babelify && npm run build:assets && rm -f lib/utils/*.mp3",
+    "build:npm": "rm -rf lib/ && npm run babelify && npm run build:assets && rm -f lib/constants/*.mp3 && rm -f lib/constants/*.png",
     "start-dev": "node dev-server/server-development",
     "start": "node dev-server/server-production",
     "lint": "eslint . --ext=js --ext=jsx",

--- a/webpack-assets.config.js
+++ b/webpack-assets.config.js
@@ -2,5 +2,5 @@ const path = require('path');
 
 module.exports = require('./make-webpack-config')({
     assetsOnly: true,
-    outputPath: path.join(__dirname, 'lib/utils')
+    outputPath: path.join(__dirname, 'lib/constants')
 });

